.github/workflows/mkosi.yml

```diff
@@ -125,7 +125,3 @@ jobs:
 
     - name: Boot ${{ matrix.distro }} QEMU
       run: timeout -k 30 10m mkosi --debug qemu
-
-    # vsock in Github Actions with qemu is broken so for now we check for failures manually.
-    - name: Check ${{ matrix.distro }} QEMU
-      run: sudo mkosi shell bash -c "[[ -e /testok ]] || { cat /failed-services; exit 1; }"
```

Comment:
Did you test this by forcing a failure in  `mkosi-check-and-shutdown`?

Reply:
Well I was first going to check whether it works at all now. But I'll add some logic to verify this works as expected